src/cms-content/learn/index.ts

```diff
@@ -8,7 +8,29 @@ import { sanitizeID } from './utils';
 type Markdown = string;
 
 /*
-  For FAQ:
+  Learn Landing page:
```

Comment:
`Markdown` can me imported from `src/cms-content/utils` now (I probably missed that on my previous PR)